protos/vehicle_connectivity.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package telemetry.vehicle_connectivity;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/teslamotors/fleet-telemetry/protos";
+
+// VehicleConnectivity represents connection status change for the vehicle
+message VehicleConnectivity {
+  string vin = 1;
+  ConnectivityEvent status = 2;
+  google.protobuf.Timestamp created_at = 3;
+}
+
+// ConnectivityEvent represents connection state of the vehicle
+enum ConnectivityEvent {
+  UNKNOWN = 0;
+  ONLINE = 1;
+  OFFLINE = 2;
```

Comment:
I wonder if we should use "CONNECTED" and "DISCONNECTED" instead of online/offline to show that this state is scoped to the fleet-telemetry connection.  E.g. the vehicle could be online but have had its fleet telemetry configuration removed.

Reply:
updated